modules/home-manager/firefox.nix

```
@@ -42,12 +58,46 @@ let
                   accentSupport = true;
                 }
                 // {
-                  force = mkOption {
-                    type = types.bool;
-                    default = false;
-                    description = "Forcibly override any existing configuration for Firefox Color.";
-                    example = true;
-                  };
+                  force = mkEnableOptionDesc "Forcibly override any existing configuration for Firefox Color.";
```

```suggestion
                  force = mkEnableOptionDesc "forcibly overriding any existing configuration for Firefox Color.";
```

So that would come out as

> Whether to enable forcibly overriding any existing configuration for Firefox Color.